pyppeteer/launcher.py

```diff
@@ -143,6 +140,11 @@ def kill(self) -> None:
         except Exception as e:
             logger.warning(f'Failed to cleanup {self.temp_dir}: {e}')
 
+    def _restore_default_signal_handlers(self) -> None:
+        for signum in self._altered_sighandlers:
```

Comment:
If you could provide some documentation as to what/why this function exists that would be great for future reference

Reply:
Tbh, I don't think I am able to provide useful documentation for this function. This patch moves the already existing code around but adds no real new functionality. I am not familiar enough with Pyppeteer to tell why these signal handlers are needed and why they must be reset.